test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -1006,6 +1006,86 @@ TEST_P(ExtAuthzLocalReplyIntegrationTest, DeniedHeaderTest) {
   cleanup();
 }
 
+TEST_P(ExtAuthzLocalReplyIntegrationTest, UpstreamTest) {
```

Comment:
would it be plausible to change the whole test to dual mode?

Reply:
This test is more or less the same as the existing downstream filter test. Are you suggesting to parameterize it?

Reply:
right, I'm saying we could run all the tests in both modes and maybe flush out other bugs